docs/netlify.toml

```diff
@@ -0,0 +1,17 @@
+[build.environment]
+  NPM_FLAGS = "--prefix=/dev/null"
+  NODE_VERSION = "16"
+
+[build]
+  publish = "docs/.vitepress/dist"
+  command = "npx pnpm i --store=node_modules/.pnpm-store && npm run docs:build"
+
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
+
+[[headers]]
+  for = "/manifest.webmanifest"
+  [headers.values]
+    Content-Type = "application/manifest+json"
```

Comment:
We have a root-level one. Is this neccessary?

Reply:
Upps, I don't see it, sorry, we can use the top one adding the webmanifest entry.

Should we use 301 instead 302 on next redirection?
/cc @patak-dev

Reply:
I prefer 302